qemu/tests/slof_device_tree.py

```diff
@@ -47,8 +47,7 @@ def check_nonexist_aliases(vm_session, devices):
         error_context.context(
             "Checking whether aliases file is indeed nonexisting", logging.info)
         if status == 0:
-            raise exceptions.TestFail(
-                "Nonexist cdrom aliases check failed.")
+            test.fail("Nonexist cdrom aliases check failed.")
```

Comment:
if there is no return value, pls update the function's docstring.

Reply:
There should be two return value for status: zero or none zero, if it is none zero, the result is expected.